docs/API.md

```diff
@@ -301,8 +324,223 @@ Rejects a pending invite for the authenticating account
 await api.rejectRealmInvitation('1234567')
 ```
 
+<details>
+<summary>Output</summary>
+
 No output
 
+</details>
+
+---
+
+### removePlayerFromRealm
+
+(realmId: string, xuid: string) => Promise\<Realm>
+
+Removes a player from the Realm
+
+```js
+await api.removePlayerFromRealm('1234567', 'a8005260a332457097a50bdbe48a9a21')
+```
+
+<details>
+<summary>Output</summary>
+
+[Realm](#realm)
+
+</details>
+
+---
+
+### resetRealm
+
+(realmId: string) => Promise\<void>
+
+Resets a Realm to its default, or brand new state
+
+```js
+await api.resetRealm('1234567')
+```
+
+<details>
+<summary>Output</summary>
+
+No output
+
+</details>
+
+---
+
+### changeRealmConfiguration
+
+(realmId: string, configuration: object) => Promise\<void>
+
+Updates a Realms configuration. This can be gamerules or general Realm settings
+
+```js
+const configuration = { description:{description: "",name: option:{slotName:"Test",pvp:true,spawnAnimals:true,spawnMonster:true,spawnNPCs:true,spawnProtection:0,commandBlocks:false,forceGameMode:false,gameMode:0,difficulty:2,worldTemplateId:-1,worldTemplateImage:"",adventureMap:false,resourcePackHash:null,incompatibilities:[],versionRef:"",versionLock:false,cheatsAllowed:true,texturePacksRequired:true,timeRequest:null,enabledPacks:{resourcePacks:[""],behaviorPacks:[""]},customGameServerGlobalProperties:null,worldSettings:{sendcommandfeedback:{type:0,value:true}commandblocksenabled:{type:0,value:true},dodaylightcycle:{type:0,value:true},randomtickspeed:{type:1,value:3},naturalregeneration:{type:0,value:true},showtags:{type:0,value:true},commandblockoutput:{type:0,value:true},dofiretick:{type:0,value:false},maxcommandchainlength:{type:1,value:65535},falldamage:{type:0,value:true},tntexplodes:{type:0,value:true},drowningdamage:{type:0,value:true},domobloot:{type:0,value:true},domobspawning:{type:0,value:true},showbordereffect:{type:0,value,:true},showdeathmessages:{type:0,value:true},respawnblocksexplode:{type:0,value:true},doweathercycle:{type:0,value:true},doentitydrops:{type:0,value:true},doimmediaterespawn:{type:0,value:true},freezedamage:{type:0,value:true},pvp:{type:0,value:true},keepinventory:{type:0,value:false},doinsomnia:{type:0,value:true},mobgriefing:{type:0,value:true},dotiledrops:{type:0,value:true},firedamage:{type:0,value:true},functioncommandlimit:{type:1,value:10000},spawnradius:{type:1,value:25},showcoordinates:{type:0,value:true}}}}}
```

Comment:
Maybe out of scope for this PR but providing some sort of config builder would make it a lot easier to interact with the API here. I.e. 

```js
const { RealmConfigBuilder } = require('prismarine-realms')

const config = new RealmConfigBuilder()
  .setName('')
  .setDescription('')
  .addWorldSettings(
    { name: 'dofiretick', value: true },
    { name: 'domobloot', value: false}
  )
  
  api.changeRealmConfiguration(config)
  ```

Reply:
Yep! I was thinking the same thing. It was really bugging me with the large JSON code block. I'll open another PR relatively soon sometime this week, or I'll just push it along with the next PR I have as a draft right now.

Reply:
You can remove this function from this PR and add it to a later one if it's not ready